src/igl/opengl/Framebuffer.cpp

```diff
@@ -879,18 +879,18 @@ void CurrentFramebuffer::bind(const RenderPassDesc& renderPass) const {
 
   // clear the buffers if we're not loading previous contents
   GLbitfield clearMask = 0;
-  if (renderPass.colorAttachments[0].loadAction != LoadAction::Load) {
+  if (renderPass.colorAttachments[0].loadAction == LoadAction::Clear) {
```

Comment:
The previous code was pretty much consistent with the comment above. What was wrong with it?

Reply:
https://github.com/facebook/igl/blob/8eb59b0f2b7d704f985c63da25b8a0ef85d55b86/src/igl/opengl/Framebuffer.cpp#L747

In **Custom**Framebuffer::bind() has the same comment, but use different conditional statements。

This leads to the situation where if the load action is LoadAction::DontCare,  glClear() will be executed in **Current**Framebuffer::bind(), but not in **Custom**Framebuffer::bind().